README.md

```diff
@@ -24,32 +25,31 @@ There is an `internal` subdirectory that contains service protocol buffers,
 plus packages required for connectivity to make API calls. App Engine apps
 should not directly import any package under `internal`.
 
-## Updating a Go App Engine app
+## Upgrading an App Engine app to the flexible environment
```

Comment:
Should we clearly state that this package doesn't work on the flexible environment?  And them point them towards the google-cloud-go package?